interchain_test.go

```diff
@@ -24,11 +24,11 @@ func TestInterchain_DuplicateChain(t *testing.T) {
 	home := t.TempDir()
 	pool, network := ibctest.DockerSetup(t)
 
-	cf := ibctest.NewBuiltinChainFactory([]ibctest.BuiltinChainFactoryEntry{
+	cf := ibctest.NewBuiltinChainFactory(zaptest.NewLogger(t), []*ibctest.ChainSpec{
```

Comment:
Might be cool to also offer a builder pattern for the `ibctest.ChainSpec`, but also might be overkill since we are just assembling a single struct.